### PR TITLE
Gulp cordova generation and cleaning

### PR DESCRIPTION
**Pull request by WalcoFPV**
The cordova related code source is now inside the `cordova` folder and the generated cordova files will be inside the `dist-cordova` folder. So, `dist-cordova` can be deleted when cleaning up after a build.
The command `gulp clean` is also fixed : `dist-cordova` folder is correctly deleted with this command.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -66,7 +67,7 @@ const SELECTED_PLATFORMS = getInputPlatforms();
 //Tasks
 //-----------------
 
-gulp.task('clean', gulp.parallel(clean_dist, clean_apps, clean_debug, clean_release));
+gulp.task('clean', gulp.parallel(clean_dist, clean_apps, clean_debug, clean_release, cordova_clean));
```
Comment (McGiverGim) [inline gulpfile.js:70]: All the other clean methods start by clean, your ends. Can you modify it to maintain the style? 
I have seen that all the Cordova methods start with Cordova but I think is better to maintain the style when possible.

File: README.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ List of possible values of `<task-name>`:
 * **release** zips up the apps into individual archives in the `./release` folder [1]. 
 
 [1] Running this task on macOS or Linux requires Wine, since it's needed to set the icon for the Windows app (build for specific platform to avoid errors).
-[2] For Android platform, **dist** task will generate the `./cordova` folder
+[2] For Android platform, **dist** task will generate folders and files in the `./dist-cordova` folder.
```
Comment (mikeller) [inline README.md:79]: Same comment here - to go with the convention for naming things in Betaflight, this should be `/cordova_dist`

**Comment by McGiverGim** [comment]
@mikeller what is the idea about nightlies of this? Generate it with azure pipelines as we do for the others tastes?

**Comment by mikeller** [comment]
@McGiverGim: Not sure if building nightlies of the Android version will be possible - for what I can tell the Android studio plugin installation cannot be scripted and requires UI interaction. But I have not looked particularly deep into this yet.

**Pull request merged**